src/Acorn/Bootstrap/HandleExceptions.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace Roots\Acorn\Bootstrap;
+
+use ErrorException;
+use Exception;
+use Illuminate\Contracts\Debug\ExceptionHandler;
+use Roots\Acorn\Application;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Debug\Exception\FatalErrorException;
+use Symfony\Component\Debug\Exception\FatalThrowableError;
+
+class HandleExceptions
+{
+    /**
+     * Reserved memory so that errors can be displayed properly on memory exhaustion.
+     *
+     * @var string
+     */
+    public static $reservedMemory;
+
+    /**
+     * The application instance.
+     *
+     * @var \Roots\Acorn\Application
+     */
+    protected $app;
+
+    /**
+     * Bootstrap the given application.
+     *
+     * @param  \Roots\Acorn\Application  $app
+     * @return void
+     */
+    public function bootstrap(Application $app)
+    {
```

Comment:
This should return early if `WP_DEBUG` is false.

I'm wondering if we should also check [`is_readable(WP_CONTENT_DIR . '/fatal-error-handler.php')`](https://github.com/WordPress/WordPress/blob/master/wp-includes/error-protection.php#L88).

Reply:
Checking for `fatal-error-handler.php` sounds sane. I will add that. I'm also considering since we want all of the exception handler to be opt-in, if I should just create a new Acorn-specific value in `app.php` to enable/disable it alongside `WP_DEBUG`?

Reply:
There's `WP_DEBUG_DISPLAY` that WordPress uses. We could require that to be true. Or we could create our own config option that derives its default value from it.

Reply:
Lets stick with `WP_DEBUG` which is our new default for `app.debug` – the handler will now bail if `debug` is set to `false` or a fatal error handler exists **unless** using acorn cli.